internal/sendconfig/common_workflows.go

```diff
@@ -94,5 +98,6 @@ func UpdateKongAdminSimple(ctx context.Context,
 		}
 	}
 
+	promMetrics.ConfigureDurationHistogram.Observe(float64(time.Since(start)))
```

Comment:
Don't cast `time.Duration` into `float`. Use `Duration` methods such as `Milliseconds()`.

Reply:
here is applying time.Since() with milliseconds. did not cast time.Duration.

Reply:
I'm not sure I understand the comment, let me try to summarize my thoughts:

`time.Since(...)` [returns](https://pkg.go.dev/time#Since) `time.Duration`.

The expression `float64(time.Since(...))` takes the result of `time.Since` which is a `time.Duration` and casts it to float64, which is invalid use of the `time.Duration` interface.

Instead of relying on the internal storage format of `time.Duration` please use its public interface (e.g. `time.Since(x).Milliseconds()`.

Reply:
👍 

This is good suggestion, but please be aware that even time.Since(x).Milliseconds() is still int64, to be acceptable by Observe, still need a minor cast.

Reply:
changed d0d14553f8bacff143d3319560b102f080d1022a